README.md

```diff
@@ -1,320 +1,77 @@
-      }
-    }
-  ],
-  "memo": ""
-}
+```sh
+make install
 ```
 
-#### Operation with nickname
-*TODO: Currently described only Hdac custom CLI. Need to add for general cases*
+The built binaries - _nodef and clif_ - will be located in your `$GOBIN`.
 
-* Query
-  * Usage: `clif hdac getbalance --wallet|--nickname|--address <owner>`
-```bash
-clif hdac getbalance --wallet walletelsa
-clif hdac getbalance --nickname princesselsa
-clif hdac getbalance --address friday1y2dx0evs5k6hxuhfrfdmm7wcwsrqr073htghpv
-
-{
-   "value": "5000000000000"
-}
-```
+## Test
 
-* transfer (send)
-  * usage: `clif hdac transfer-to <recipient_address_or_nickname> <amount> <fee> <gas-price> --address|--wallet|--nickname <from>`
-```bash
-clif hdac transfer-to sisteranna 1000000 100000000 20000000 --wallet walletelsa
-clif hdac transfer-to sisteranna 1000000 100000000 20000000 --nickname princesselsa
-clif hdac transfer-to sisteranna 1000000 100000000 20000000 --address friday1y2dx0evs5k6hxuhfrfdmm7wcwsrqr073htghpv
+You should launch execution engine grpc server first.
 
-...
-confirm transaction before signing and broadcasting [y/N]: y
-Password to sign with 'walletelsa': # input your password
-{
-  "height": "0",
-  "txhash": "141F12A891659F52B055EF7F701B1D406E5F1721CE929630CC5CE3CE0C4C8718",
-  "raw_log": "[{\"msg_index\":0,\"success\":true,\"log\":\"\",\"events\":[{\"type\":\"message\",\"attributes\":[{\"key\":\"action\",\"value\":\"executionengine\"}]}]}]",
-  "logs": [
-    {
-      "msg_index": 0,
-      "success": true,
-      "log": "",
-      "events": [
-        {
-          "type": "message",
-          "attributes": [
-            {
-              "key": "action",
-              "value": "executionengine"
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
-```
-* bond
-  * usage: `clif hdac bond --wallet|--address|--nickname <owner> <amount> <fee> <gas-price>`
 ```sh
-clif hdac bond --wallet walletelsa 1000000 100000000 30000000
-clif hdac bond --nickname princesselsa 1000000 100000000 30000000
-clif hdac bond --address friday1y2dx0evs5k6hxuhfrfdmm7wcwsrqr073htghpv 1000000 100000000 30000000
-
-confirm transaction before signing and broadcasting [y/N]: y
-Password to sign with 'walletelsa':
-{
-  "height": "0",
-  "txhash": "22DF1E0D8D9EB8BE2B5F50995C6FC0AB20E34715875A9F9856A9466A8C406807",
-  "raw_log": "[{\"msg_index\":0,\"success\":true,\"log\":\"\",\"events\":[{\"type\":\"message\",\"attributes\":[{\"key\":\"action\",\"value\":\"executionengine\"}]}]}]",
-  "logs": [
-    {
-      "msg_index": 0,
-      "success": true,
-      "log": "",
-      "events": [
-        {
-          "type": "message",
-          "attributes": [
-            {
-              "key": "action",
-              "value": "executionengine"
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
+cd friday
+./CasperLabs/execution-engine/target/release/casperlabs-engine-grpc-server $HOME/.casperlabs/.casper-node.sock$
```

Comment:
Is `$` typo? appended at the socket file path.

Reply:
fixed it, bro :)